cmd/wardenkms/wardenkms.go

```diff
@@ -56,18 +77,26 @@ func main() {
 		return
 	}
 
+	grpcConfigs, err := mapGrpcConfig(cfg.GRPCURLs)
```

Comment:
since wardenkms is an user of the keychain-sdk, having to do this sort of mapping "manually" shows that the configuration of the keychain-sdk is not very ergonomic

i.e. keychain-sdk is not a library that I would enjoy using

i think we can explore different configuration patterns such as the option pattern maybe, or even treating the URL string as "special", like if it starts with "http://" or "tcp://" turn on the insecure option, while if it starts with "https://" turn insecure off

but, if you have better ideas please share them!